src/animl/plot_boxes.py

```diff
@@ -0,0 +1,63 @@
+"""
+Module: animl.plot_boxes
+Functionality to draw bounding boxes and labels provided image DataFrame.
+"""
+
+import cv2
+
+
+def draw_bounding_boxes(
```

Comment:
I'm assuming the arguments are on their own lines due to linting but I find this difficult to read/visually unappealing

Reply:
The way this particular formatting has been done does strike my eye and is, actually, kind of unconventional (which is probably why my eye sees it as unappealing) so we should probably make it more conventional. However, it is not unusual and, frequently necessary, to have arguments spread across lines in Python.  I actually find it much more visually appealing in a lot of instances to having the function arguments going off far to the right and, especially if wrapped by the text editor, all so tightly clumped together.  

It doesn't have to be one argument per a line, though.

Certainly, when possible, variable names can be chosen to be shorter (or variables can be collected into meaningful aggregations such as dictionaries or NamedTuples and handed over to the function together) to make a function signature be on one line but one doesn't want to do that at the expense of having good variable names that explain themselves well or creating arbitrary conglomerations.

Anyway, just inserting some thoughts and am game for whatever we want to do here in this particular instance.

Reply:
I said that the formatting was kind of unconventional because I myself always have the first argument on the same line as the function name and am used to seeing that in my own code and certainly also a lot of other code.  But I doubted that assertion and went and looked for examples that use the formatting that Nidhi has used here and found plenty.  Including, for instance, in `pandas` e.g. this example:

`def hist_frame(
    data: "DataFrame",
    column: Union[Label, Sequence[Label]] = None,
    by=None,
    grid: bool = True,
    xlabelsize: Optional[int] = None,
    xrot: Optional[float] = None,
    ylabelsize: Optional[int] = None,
    yrot: Optional[float] = None,
    ax=None,
    sharex: bool = False,
    sharey: bool = False,
    figsize: Optional[Tuple[int, int]] = None,
    layout: Optional[Tuple[int, int]] = None,
    bins: Union[int, Sequence[int]] = 10,
    backend: Optional[str] = None,
    legend: bool = False,
    **kwargs,
):`

Reply:
Just to mark what we discussed in slack, the proper format would be 
def draw_bounding_boxes(row, box_number, image_output_path):
    # pylint: disable=R0914
    """  DOCSTRING """

Please resubmit and I will merge.  Thanks!